### ISSUE TITLE
[BUG]Not identifying devices correctly?

### ISSUE THREAD

**Issue by mmaster23** [yllibed/Zigbee2MqttAssistant#129]
# Describe the bug
Z2MA not dectecing the devices correctly? Z2M seems to work just fine as well as the Zigbee stick itself yet the assistant isn't working properly. Is this normal?

**Expected behavior**
Was expecting overview like the screenshots on the site shows.

## Steps I took to try resolve the problem
Tried multiple versions, running this in Docker.

## Screenshots
![Dev1](https://user-images.githubusercontent.com/21027839/69812176-d9f83600-11ef-11ea-958a-13f4b020f569.PNG)
![Dev2](https://user-images.githubusercontent.com/21027839/69812181-dbc1f980-11ef-11ea-962e-9e9f1f741ca7.PNG)
![Dev3](https://user-images.githubusercontent.com/21027839/69812184-dd8bbd00-11ef-11ea-89c8-2b32cc8c584c.PNG)

## Installation
 - Version of Zigbee2Mqtt: 1.7.1
 - Version of Zigbee2MqttAssistant: 0.3.105
 - Installation type (HASS.IO, Docker...): Docker
 - Docker version + CPU type (amd64, arm arm64...): amd64 
Server:
 Engine:
  Version:          18.09.6
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.11
  Git commit:       1d8275b
  Built:            Fri Sep  6 02:51:05 2019
  OS/Arch:          linux/amd64
  Experimental:     false

**Comment by mmaster23** [comment]
Weird enough it started working when I turned on the Homeassistant option in z2m config.. Is this by design?

**Comment by FutureCow** [comment]
Yes, this is by design. But for me this is not working well. When HA is enabled, all my switches sending the command twice!

Would it be possible to make Z2MA work without HA enabled?

**Comment by carldebilly** [comment]
@FutureCow Yes it's possible.  Should work and since 1.7.x, it's possible to remove this requirement because the "get devices" is now returing enough information.

I think I can remove this requirement in the documentation.

**Comment by FutureCow** [comment]
@carldebilly thanks for the reply, you mean that this will be possible in a future update? I'm running latest z2m and z2ma, but it's not working right now.

**Comment by carldebilly** [comment]
Yes. It's not currently possible, but I'll have this behavior to activate itself only on older version of Z2M.

Thanks for reporting!

**Comment by carldebilly** [comment]
@all-contributors please add FutureCow to bug
